CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
```

Comment:
I have never worked with CHANGELOG.md files.  What benefit does this provide the project?  Thanks!

Reply:
Great question. A [Changelog](https://keepachangelog.com/en/1.0.0/) gives users of the your project notice of what changed. It serves as a log for all you changes so that a user may easily see what has changed with the project. 

By following [SemVer](https://semver.org/spec/v2.0.0.html) (x.y.z), users are able to tell wether a release provides a Bug fix (z), a new feature (y), or if there is a breaking change (x). These changes/additions are then called out in your changelog for easy reference so that users don't have to go digging through commits. You can find more in-depth details in the links I provided.